src/ucp/core/ucp_mm.c

```diff
@@ -630,6 +630,34 @@ ucp_memh_alloc(ucp_context_h context, void *address, size_t length,
     return status;
 }
 
+static ucs_status_t
+ucp_memh_reg(ucp_context_h context, void *address, size_t length,
```

Comment:
Maybe pick some better name for this functons. It's hard to tell the difference between `ucp_memh_reg`, `ucp_memh_register` and `ucp_memh_init_uct_reg`